src/App.css

```diff
@@ -1,51 +1,39 @@
-.App {
-  text-align: center;
-}
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500&family=Nunito:wght@200;400&family=Roboto&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Allerta+Stencil&family=Heebo:wght@400;500;600&family=Noto+Sans&family=Nunito:ital,wght@0,300;1,300&family=Playfair+Display:wght@700;800&family=Poppins:wght@200;300;400&family=Public+Sans:wght@200&display=swap");
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+:root {
+  --bg-color: #eee9f5;
```

Comment:
Suggestion- since you are using tailwind, you can add these values as util classes at the utilities layer and use them as classes anywhere in your app. This way you can avoid writing CSS separately everytime you want to use these variables.